src/components/RouteSchedule/RouteSchedule.js

```diff
@@ -169,18 +200,26 @@ const renderHeader = lineInfos => (
     </span>
     <div className="rt-route-title">
       <span className="rt-route-name">{lineInfos.destination}</span>
-      <span>{lineInfos.longName}</span>
+      <span>
+        {`${lineInfos.longName} ${
+          lineInfos.routeIdentifier.split('.')
+            ? `(${lineInfos.routeIdentifier.split('.')[0]})`
+            : ''
```

Comment:
if routIdentifier exists and it's a string, result of the split will always be an array with a 0 index.  So you can display the result directly
```
{`${lineInfos.longName} ${lineInfos.routeIdentifier.split('.')[0]}`}
```